packages/cms-cli/commands/create.js

```diff
@@ -111,7 +111,7 @@ function configureCreateCommand(program) {
           type
         )
       ) {
-        dest = name;
+        dest = name || type;
```

Comment:
I think this makes sense for `website-theme` and `react-app` but not for `function`. What if we did it just for `website-theme` and `react-app`?

Reply:
I updated this. I am curious if the switch pattern works or if it would make sense to stick with the `[].includes()` pattern?

Reply:
The switch works fine. Another option would be to have some of config object that could be used to look up the config based on the type. If the different types continue to expand in complexity, it may make sense to go in that direction.